python/composio_swe/benchmark/run_evaluation.py

```diff
@@ -48,7 +48,7 @@ def filter_from_repo_name(curr_dataset, repo_name):
 
 
 def get_issues_dataset():
-    test_dataset = load_dataset("princeton-nlp/SWE-bench_Lite", split="test[1:50]")
+    test_dataset = load_dataset("princeton-nlp/SWE-bench_Lite", split="test[210:300]")
```

Comment:
pls use it from the constants file directky, later i am planning to use a config.yaml so all this config should come from config level